chapters/04-function-development.Rmd

```diff
@@ -13,10 +13,36 @@
 - Function documentation (with roxygen2), part 1
 - To use `@examples` to help with creating function
 
-
-
 ## Final exercise
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. Add a `FIXME()` function to your FIXME package
+    
+    The following code snippet produces ... FIXME.  
+    ```{r, eval = FALSE}
+    # A line or two of R code that does something useful
+    FIXME
+    ```
+    Turn it into a function called `FIXME()` that takes argument for       `FIXME` and `FIXME` and include it in your FIXME package.
```

Comment:
I like this a lot - providing a regular code snippet to turn into a function. This mimics the natural process of code development.

Reply:
Do you want to add instructions to create function documentation explicitly? It's implied at the end of question 2, just wondering if it should be part of question 1 as well.

Reply:
Good point, I've added an explicit step.